website/content/docs/agent/telemetry.mdx

```diff
@@ -684,3 +684,22 @@ agent. The table below describes the additional metrics exported by the proxy.
 | `consul.proxy.web.upstream.conns`   | Shows the current number of connections open from a proxy instance to an upstream. Where supported a `src` label is added indicating the service name the proxy represents, and a `dst` label is added indicating the service name the upstream is connecting to. | connections | gauge   |
 | `consul.proxy.web.inbound.rx_bytes` | Increments by the number of bytes received from an upstream connection. Where supported a `src` label is added indicating the service name the proxy represents, and a `dst` label is added indicating the service name the upstream is connecting to.       | bytes       | counter |
 | `consul.proxy.web.inbound.tx_bytes` | Increments by the number of bytes transferred to an upstream connection. Where supported a `src` label is added indicating the service name the proxy represents, and a `dst` label is added indicating the service name the upstream is connecting to.      | bytes       | counter |
+
+## Peering metrics
+
+** Requirements: **
+* Consul 1.13.0+
+
+Consul 1.13 introduced [cluster peering](/docs/connect/cluster-peering). These metrics are only emitted by the leader server.
```

Comment:
```suggestion
[Cluster peering](/docs/connect/cluster-peering) refers enabling communication between Consul clusters through a peer connection, as opposed to a federated connection. Consul collects metrics that describe the number of services exported to a peered cluster. Peering metrics are only emitted by the leader server.  
```
1. The Consul version is already stated in the requirements
2. Added a statement about what cluster peering is so readers would have a little more context without having to follow the link.
3. It appears that there is only one metric for now that describes the number of exported services, correct?